tests/test_cache.py

```diff
@@ -282,3 +419,19 @@ def get_cache_expire_string_year(self):
     @Cache(key="test_cache_expire_string_year", invalidate=True)
     def clear_cache_expire_string_year(self):
         pass
+
+    @Cache(key="test_cache_empty", expire=timedelta(seconds=2))
+    def get_cache_empty(self, param):
+        return param

```

Comment:
the name `get_cache_empty` is not clear. 
suggestion:
`get_cache_empty_results`